### PR TITLE
allow php7

### PR DESCRIPTION
**Pull request by lekoala**
update sdk to 1.7 and use the same php requirements

### CODE REVIEW

**Comment by zapur1** [comment]
bump

File: composer.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
         "silverstripe/sentry": "*"
     },
     "require": {
-        "php": ">=5.3.3,<7",
-        "sentry/sentry": "1.6.2",
+        "php": "^5.3|^7.0",
```
[the remark this answers is not in the document]
Comment (phptek) [reply composer.json]: @lekoala Bump

[the remark this answers is not in the document]
Comment (lekoala) [reply composer.json]: sure that make sense, do you need anything from my side?

[the remark this answers is not in the document]
Comment (phptek) [reply composer.json]: Nah, I'll merge it and patch it up when I get a moment. Cheers!

**Pull request merged**